doc/api/zlib.md

```diff
@@ -107,6 +107,8 @@ const http = require('http');
 const fs = require('fs');
 http.createServer((request, response) => {
   const raw = fs.createReadStream('index.html');
+  // instruct the proxy to store both a compressed and uncompressed version of the resource
```

Comment:
Hi, @mukulkhanna! Welcome and thanks for the pull request! This line above is (probably) going to cause the linter to complain. Could you wrap the comment at 80 characters and capitalize the first letter of the first word? (Might as well put a period/full-stop at the end while you're at it.)

Reply:
Oh, and you can check the linting with `make lint` (or `vcbuild lint` on Windows).

Reply:
Hi @Trott.
Sure, will do asap, thank you.